src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/component/LocationMapper.kt

```diff
@@ -19,4 +19,8 @@ class LocationMapper {
   fun mapToCurrentContextPathAsString(path: String, id: UUID): UriComponents {
     return ServletUriComponentsBuilder.fromCurrentContextPath().path(path).buildAndExpand(id)
   }
+
+  fun mapToCurrentContextPathAsAppointment(path: String, id: UUID, sessionsNumber: Int): UriComponents {
+    return ServletUriComponentsBuilder.fromCurrentContextPath().path(path).buildAndExpand(id, id)
```

Comment:
there's a typo here, we are expanding `id` twice. a unit test would catch this. 

i also think the method name here is really unhelpful. i wouldn't be able to guess what this method does from the name. same with the method above.

Reply:
this has been addressed in a separate change so is no longer an issue